test_results_win64.csv

```diff
@@ -87,7 +87,7 @@ Dish Stirling,All Equity Partnership Flip,207236624,18.860062,20.753241,;
 Dish Stirling,Sale Leaseback,207236624,19.091169,19.771194,; 
 Dish Stirling,LCOE Calculator,207236624,NA,NA,; 
 Dish Stirling,None,207236624,NA,NA,; 
-Generic CSP System,Commercial,573450304,3.715794,NA,; Notice: time 0.00 { Generic solar model 2 }:\ The interpolation code must be 1 (interpolate) or 2 (nearest neighbor)The input value was 0, so it was reset to 1
+Generic CSP System,Commercial,574225408,3.711204,NA,; Notice: time 0.00 { Generic solar model 2 }:\ The interpolation code must be 1 (interpolate) or 2 (nearest neighbor)The input value was 0, so it was reset to 1
```

Comment:
@cpaulgilman I was mistaken in the SAM team meeting yesterday, it looks like the actual results match the _old_ values for this line:

```
Large change in ANNUAL_OUTPUT for Generic CSP System Commercial. Expected 574225408 / Actual: 573450304
```

Can you confirm that you had the latest patch ssc checked out when you ran test_script_ow.lk? If so, I'm ok to merge this into patch and fix this on the patch branch.

Reply:
Yes, I ran the script using the latest patch version.